src/sdk/namespace_fs.js

```diff
@@ -1759,6 +1800,213 @@ class NamespaceFS {
             throw new RpcError('BAD_REQUEST', 'Bad Request');
         }
     }
+
+    // delete latest version - 
```

Comment:
Do we need this aggregated comment? it seems both function have a comment of their own about what they do, I think it's long and unneeded, maybe take this comment from here and copy to the function as it seems this ones are more informative.

Reply:
maybe change to comment about this function gets a single key w/o version_id and delete it. Not suer if we need this function anyway, do we use it more than once? plan to use it more than once?

Reply:
I removed it, for now, we might need it when we will start working on suspended versioning flows